client/src/models/index.ts

```diff
@@ -18,12 +18,25 @@ export type ServerMessage = {
   user: string;
   embed?: Embed;
   content: string;
-  mentions: string[];
+  sentTimestamp: number;
+  mentions: Array<string>;
+};
+
+export type ServerEditMessage = {
```

Comment:
is it possible to extend message create from message edit?

Reply:
sure, ill have a look